traffic_server/datadog_checks/traffic_server/metrics.py

```diff
@@ -0,0 +1,1850 @@
+        'name': 'process.cache.volume.hdr_marshals',
+        'tags': ('cache_volume',),
+        'method': 'gauge',
+    },
+    {
+        'regex': r'proxy.process.cache.(.*?)\.hdr_marshal_bytes',
+        'name': 'process.cache.volume.hdr_marshal_bytes',
+        'tags': ('cache_volume',),
+        'method': 'gauge',
+    },
+    {
+        'regex': r'proxy.process.cache.(.*?)\.gc_bytes_evacuated',
+        'name': 'process.cache.volume.gc_bytes_evacuated',
+        'tags': ('cache_volume',),
+        'method': 'gauge',
+    },
+    {
+        'regex': r'proxy.process.cache.(.*?)\.gc_frags_evacuated',
+        'name': 'process.cache.volume.gc_frags_evacuated',
+        'tags': ('cache_volume',),
+        'method': 'gauge',
+    },
+    {
+        'regex': r'proxy.process.cache.(.*?)\.sync.count',
+        'name': 'process.cache.volume.sync.count',
+        'tags': ('cache_volume',),
+        'method': 'gauge',
+    },
+    {
+        'regex': r'proxy.process.cache.(.*?)\.sync.bytes',
+        'name': 'process.cache.volume.sync.bytes',
+        'tags': ('cache_volume',),
+        'method': 'gauge',
+    },
+    {
+        'regex': r'proxy.process.cache.(.*?)\.sync.time',
+        'name': 'process.cache.volume.sync.time',
+        'tags': ('cache_volume',),
+        'method': 'gauge',
+    },
+    {
+        'regex': r'proxy.process.cache.(.*?)\.span.errors.read',
+        'name': 'process.cache.volume.span.errors.read',
+        'tags': ('cache_volume',),
+        'method': 'monotonic_count',
+    },
+    {
+        'regex': r'proxy.process.cache.(.*?)\.span.errors.write',
+        'name': 'process.cache.volume.span.errors.write',
+        'tags': ('cache_volume',),
+        'method': 'monotonic_count',
+    },
+    {
+        'regex': r'proxy.process.cache.(.*?)\.span.failing',
+        'name': 'process.cache.volume.span.failing',
+        'tags': ('cache_volume',),
+        'method': 'gauge',
+    },
+    {
+        'regex': r'proxy.process.cache.(.*?)\.span.offline',
+        'name': 'process.cache.volume.span.offline',
+        'tags': ('cache_volume',),
+        'method': 'gauge',
+    },
+    {
+        'regex': r'proxy.process.cache.(.*?)\.span.online',
+        'name': 'process.cache.volume.span.online',
+        'tags': ('cache_volume',),
+        'method': 'gauge',
+    },
+]
+
+
+def build_metric(metric_name, logger):
+    # type: (str, Any) -> Tuple[Optional[str], Optional[List[str]]]
+    """
+    proxy.node.restarts.proxy.restart_count
+    proxy.process.cache.volume_1.span.offline
+    proxy.process.http.101_responses
+    """
+    additional_tags = []
+    name = metric_name
+    found = False
+
+    if SIMPLE_METRICS.get(metric_name):
+        found = True
+        metric_mapping = SIMPLE_METRICS.get(metric_name)
+        name = metric_mapping.get("name")
+        method = metric_mapping.get("method")
+
+    else:
+        found = False
+        for regex in REGEX_METRICS:
```

Comment:
You can remove `found` by doing `for...else`

Reply:
Good idea! I also updated some of the logic like creating a default method value